_posts/2015-07-29-nodebots-writeup.md

```diff
@@ -0,0 +1,57 @@
+---
+published: true
+title: NodeBots Writeup
+layout: post
+date: 2015-07-29
+author: "[Rob Shepherd](https://twitter.com/95point2)"
+---
+
+Last saturday saw International Nodebots Day, and another [Summer of Hacks](http://summerofhacks.io) event from JSOxford.
+
+The guys put together a great event in partnership with Gordon from Pur3.
+Gordon is the talented inventor of the Espurino family of javascript powered microcontrollers and had previously given a talk to the group about the boards, so most people were already juiced up to try it out.
+
+![Table of stuff](/img/Nodebots-table-of-stuff.jpg)
+{: .text-center }
+
+A heap of wires, batteries, servos and other electronics had been duly arranged and Gordon had even gone to the supreme effort of developing a printed circuit board to form the chassis of the mobile robot
+and had instructions prepared to give us a head start of assembling our soon-to-be planet destroying synthetic lifeform underlings!
+
+Motivation to begin soldering was high and one could feel the anticipation of the day's creativity in the air, and as the crumbs of french pastries still lingered on moustaches, the hardware hacking fury began.
+
+Surges of caffeine countered with the absolute need of a steady hand for soldering surface mount components onto the board but the group tore through the hardware assembly eager to get to promised land of embedded Javascript control of their new robotic creations.
+Early finishers eased out of hardware assembly and hit the whiteboards to sketch out the software needed for world domination via robots equipped with light sensors and servos for forward and backward motion and possibly a bluetooth transceiver.
+
+![Hard at work on NodeBots](/img/Nodebots-hard-at-work.jpg)
+{: .text-center }
+
+As delicious as the Burritos were, lunch was a mere footnote of the day as teams inhaled the mexican food, quaffed a beer and rushed back to the lab to focus on the hacking.
+
+Mid afternoon saw a flurry of creations being given the spark of life in the modern-day primordial soup of burrito-stained wiring, hot glue residue and embedded Javascript and an veritable army of imposing artificial beings were sent forth to spin arbitrarily and crash into skirting boards as their human overlords scratched their beards and contemplated the bewildering complexity of providing motion through esoteric pulse width modulation schemes expressed in javascript functions.
+
+![Spinning Bot](/img/Nodebots-spin.gif)
+{: .text-center }
+
+Thereafter, designs were iterated and robotic displays such as "light following" plus the odd whiteboard "what could have been" presentations were duly impressed upon the crowd.
```

Comment:
Since I don't have push access (and I really don't want to go making a pull request for a pull request - that would just blow my mind), could someone add the link to my video on Twitter?
`[light following](https://twitter.com/danielthepope/status/624969982934392832)`
Ta :)